lib/model.js

```diff
@@ -4880,6 +4880,33 @@ Model.compile = function compile(name, schema, collectionName, connection, base)
   return model;
 };
 
+Model.clientEncryption = function clientEncryption() {
+  const ClientEncryption = driver.get().ClientEncryption;
```

Comment:
You should be able to use `this.base.driver.get()` instead of `require('./driver')` directly, see `Model.bulkWrite()` for an example.

Technically drivers are scoped to Mongoose instances, so you can have multiple Mongoose instances using different drivers in one process.

Reply:
Didn't realize that - the getter/setters in `driver.js` seemed to be setting a module-scoped variable.  

Fixed!

Reply:
Yeah the module scoped variable is the default driver, but individual Mongoose instances can have their own driver which overrides the default.